meow/eme/common.py

```diff
@@ -27,10 +28,11 @@ def compute_interface_s_matrix(
     # extra phase correction.
 
     # ignoring the phase seems to corresponds best with lumerical.
-    O_LL = np.abs(O_LL)
-    O_RR = np.abs(O_RR)
+    #O_LL = np.abs(O_LL)
+    #O_RR = np.abs(O_RR)
 
     # alternative phase correction (probably worth testing this out)
+    # Question: is this not just a conjugation?
```

Comment:
I think this is just equivalent to `abs` as you're multiplying with the inverse of the phase of the original hence removing it. Conjugating would involve a factor -2j in stead of a factor -1j.

Reply:
True! My oversight